### ISSUE TITLE
Make `me` able to sort words between verbs and nouns

### ISSUE THREAD

**Issue by raphydaphy** [raphydaphy/Q-Operating-System#149]

**Comment by Barteks2x** [comment]
Should it be a command? Why would you need that?

**Comment by raphydaphy** [comment]
For `me` the thingy that I'm working on for a virtual assistant in Q OS, it takes a few steps to get the answer and it does it by machine learning. The first step of that is sorting words between there types of words. I've started making this in `sort.c` located in `kernel/apps/me/sort.c` and here is a direct copy-paste of a comment block in it.

    // Here are the basic category numbers, for more information look on the Q OS Wiki for Me
    // Link: https://github.com/raphydaphy/Q-Operating-System/wiki/Me
    
    // 0 = unknown                      The kind of this word is unknown or unspecified
    // 1 = noun                         A noun is a person, place, thing, quality or act word
    // 2 = verb                         Verbs are action/existence words that say what a noun does
    // 3 = adjective                    An adjective is a word that describes a noun
    // 4 = Adverb                       An adverb descries a verb, adjective or adverb. It often ends in 'ly'
    // 5 = Interjection                 An interjection is an outcry/ sudden utterance. Usally the start of a sentence
    // 6 = Preposition                  A preposition describes the relationship between a noun and another noun (can also be verb/adverb)
    // 7 = Conjunction                  A conjunction is a word that joins together multiple thoughts
    // 8 = Pronoun                      A pronoun replaces a noun/noun phase that is understood from context

Basically, `me` needs to know what kind of word each word in your question is to try to get a contextual meaning of the entire sentence.

I'm working on me strictly by myself atm btw, because it's not a entirely essential part of Q OS but will be very powerful if complete. Basically, things like multitasking, filesystem and `calc` are more important to the near future of Q OS but this is more important to the far future...

**Comment by Barteks2x** [comment]
The problem with such kind of thing is that it's very specific to the language and will definitely make internationalization a lot harder. And it definitely shouldn't be implemented as kernel function. But to be able to to things outside of kernel we need working multitasking, enable long mode, (use cross compiler?), and get userspace working (ring 3)

**Comment by Barteks2x** [comment]
Also, you seem to use numbers (ints) to describe category. Use enum for that (that's what enum is for...).

**Comment by raphydaphy** [comment]
im using numbers to describe catagory because there are sub catagories,

idk if you can have nested enums...

**Comment by Barteks2x** [comment]
you can use different enum for subcategory.

**Comment by raphydaphy** [comment]
ok den
